UI/Result/index.js

```diff
@@ -1,29 +1,40 @@
-import React from "react"
+import React, {useEffect} from "react"
 import resultStyles from "./result.module.css"
-import { GetAttributesFromCard } from "../../src/utils/helpers"
-import { PLAYER_USER } from "../../src/pages/game"
-import { Card } from "../Card"
-import { Button } from "../Button"
+import {GetAttributesFromCard} from "../../src/utils/helpers"
+import {PLAYER_COMPUTER, PLAYER_USER} from "../../src/pages/game"
+import {Card} from "../Card"
+import {Button} from "../Button"
 
 export function Result({
-   computerName,
-   usersTurnCard,
-   computersTurnCard,
-   selectedAttribute,
-   incrementTurnCount,
-   winner,
-}) {
+                          computerName,
+                          usersTurnCard,
+                          computersTurnCard,
+                          selectedAttribute,
+                          incrementTurnCount,
+                          winner,
+                       }) {
 
    const playerWon = winner === PLAYER_USER
 
+   const message = () => {
+      if (winner === PLAYER_USER) {
+         return 'Round Won!!'
+      }
+      if (winner === PLAYER_COMPUTER) {
+         return 'Round Lost!!'
+      }
+
+      return 'Draw';
+   }
+
```

Comment:
`message` may just as well be a constant string, rather than a function, but this is a minor change

Reply:
Whats the syntax to get that to work with the nested if? I just made it a function as thats the only way i Knew :P

Reply:
Oh gawd yeah I see what you mean! 🙈 

You could use a `let`?

```js
let message = 'Draw';
if (winner === PLAYER_USER) {
  message = 'Round Won!!';
}
if (winner === PLAYER_COMPUTER) {
  message = 'Round Lost!!';
}
```

But I'm not convinced that's a whole lot better! Up to you 😄